### PR TITLE
Appman-876 Initial setup to start the import of the data

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

File: src/SFA.DAS.ReferenceData.Jobs/readme.md
Comment (cofaulco) [inline src/SFA.DAS.ReferenceData.Jobs/readme.md:1]: can we add a proper readme for the application please?

File: src/SFA.DAS.ReferenceData.Jobs/Interfaces/IApimClient.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.ReferenceData.Jobs.Interfaces
+{
+    public interface IApimClient
```
Comment (cofaulco) [inline src/SFA.DAS.ReferenceData.Jobs/Interfaces/IApimClient.cs]: I believe in other solutions we refer to 'OuterApi' as opposed to APIM. Could we maintain the consistency there?

**Review by cofaulco — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,165 +1,97 @@
...
-    - Aborted - all retrieved messages will be placed back on the queue they were received from
-    - Replayed - messages will be moved back onto the original processing queue so that they can be processed again
-    - Deleted - messages will be removed and will be no longer available for processing
-```
+This is part of the Reference Data project, and is solely used to start the import process for each of the external data sources.
 
 ## How It Works
 
-_Add a description of how the project works technically, this should give new developers an insight into the how the project hangs together, the core concepts in-use and the high-level features that it provides_
+A NServiceBus timer periodically starts the import process, this import process calls the outer API which in turn fetches the external data and populates it in the appropriate databases.
```
Comment (cofaulco) [inline README.md]: the timer trigger is not NServicebus related

File: README.md [hunk 3]
```diff
@@ -1,165 +1,97 @@
...
 ```
-* .NetCore 3.1
-* Azure Functions V3
-* CosmosDB
-* REDIS
-* NLog
+* .Net 6.0
+* Azure Functions V4
 * Azure Table Storage
+* NServiceBus
```
Comment (cofaulco) [inline README.md]: don't think we have anything NServicebus related

File: README.md [hunk 4]
```diff
@@ -1,165 +1,97 @@
...
 _List the key technologies in-use in the project. This will give an indication as to the skill set required to understand and contribute to the project_
 
 _For Example_
 ```
-* .NetCore 3.1
-* Azure Functions V3
-* CosmosDB
-* REDIS
-* NLog
+* .Net 6.0
```
Comment (cofaulco) [inline README.md]: Net 8.0?

File: src/SFA.DAS.ReferenceData.Jobs/SFA.DAS.ReferenceData.Jobs.csproj [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    <PackageReference Include="System.Drawing.Common" Version="8.0.3" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
+    <PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="host.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="local.settings.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```
Comment (cofaulco) [inline src/SFA.DAS.ReferenceData.Jobs/SFA.DAS.ReferenceData.Jobs.csproj]: do we need to explicitly make sure this isn't copied to the publish directory?

File: src/SFA.DAS.ReferenceData.Jobs/readme.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# TimerTrigger - C<span>#</span>
```
Comment (cofaulco) [inline src/SFA.DAS.ReferenceData.Jobs/readme.md]: second Readme?

**Review by cofaulco — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Pauljgraham) [reply README.md]: I'm always amazed, how stupid I can be at times. Of course it's not related to NSB,

File: README.md (hunk 4 above)
Comment (Pauljgraham) [reply README.md]: working for too long on .NET 6 upgrades

File: src/SFA.DAS.ReferenceData.Jobs/SFA.DAS.ReferenceData.Jobs.csproj (hunk 5 above)
Comment (Pauljgraham) [reply src/SFA.DAS.ReferenceData.Jobs/SFA.DAS.ReferenceData.Jobs.csproj]: No, quite the opposite

File: src/SFA.DAS.ReferenceData.Jobs/readme.md (hunk 6 above)
Comment (Pauljgraham) [reply src/SFA.DAS.ReferenceData.Jobs/readme.md]: deleted

**Review by cofaulco — APPROVED** [review]

**Pull request merged**